src/components/Trade/Components/TradeQuotes/TradeQuote.tsx

```diff
@@ -123,13 +164,13 @@ export const TradeQuoteLoaded: React.FC<TradeQuoteLoadedProps> = ({
           <RawText>{protocol}</RawText>
         </Flex>
         <Amount.Crypto
-          value={quoteAmountCryptoPrecision}
-          symbol={asset?.symbol ?? ''}
+          value={amountEntered && !negativeRatio ? totalReceiveAmountCryptoPrecision : '0'}
```

Comment:
There is some repetition in `amountEntered && !negativeRatio` checks, do we want to abstract these into a `isValidAmount` kind of naming?

Reply:
Resolved in https://github.com/shapeshift/web/pull/4079/commits/0543329e80fb6c0944b4790d45acc67db58a2064